CRM/Utils/Check/Component/Env.php

```diff
@@ -453,16 +453,16 @@ public function checkDirsWritable() {
     if (!empty($notWritable)) {
       $messages[] = new CRM_Utils_Check_Message(
         __FUNCTION__,
-        ts('The %1 is not writable.  Please check your file permissions.', array(
+        ts('The %1 is not writable. Writing to Settings file has been disabled by admin. Please check your file permissions.', array(
```

Comment:
I agree with @eileenmcnaughton that this text "writing to Settings file has been disabled by admin" is confusing.  This warning can appear in several situations, but I don't think any of them involve writing to a settings file.